requirements.txt

```diff
@@ -23,7 +23,7 @@ docutils==0.15.2          # via botocore
 ecdsa==0.15               # via python-jose
 enum34==1.1.6             # via cryptography, grpcio, h2
 firebase-admin==3.2.1
-futures==3.3.0            # via google-api-core, grpcio, s3transfer
+futures==3.1.1            # via google-api-core, grpcio, s3transfer
```

Comment:
not sure why, but I was getting build errors (availability) trying to use 3.3.

Reply:
Huh if anything I'd think 3.1.1 would have availability errors because it was [yanked](https://pypi.org/project/futures/#history), but it still works. pip reports in ci it was yanked for a minor issue:
```
WARNING: The candidate selected for download or install is a yanked version: 'futures' candidate (version 3.1.1 at https://files.pythonhosted.org/packages/a6/1c/72a18c8c7502ee1b38a604a5c5243aa8c2a64f4bba4e6631b1b8972235dd/futures-3.1.1-py2-none-any.whl#sha256=c4884a65654a7c45435063e14ae85280eb1f111d94e542396717ba9828c4337f (from https://pypi.org/simple/futures/))
Reason for being yanked: Does not declare incompatibility with Python 3
Collecting futures==3.1.1
  Downloading futures-3.1.1-py2-none-any.whl (14 kB)
```